hilti/runtime/src/tests/regexp.cc

```diff
@@ -12,10 +13,17 @@
 using namespace hilti::rt;
 using namespace hilti::rt::bytes::literals;
 
+namespace hilti::rt::detail::adl {
+template<typename A, typename B>
+std::string to_string(const std::tuple<A, B>& xs, tag /*unused*/) {
```

Comment:
Hmmm .. I'm surprised we need this, tuples should have a to_string() already. Also should output them as `(..)`.